test/demuxer_test.exs

```diff
@@ -0,0 +1,139 @@
+defmodule Membrane.WebM.DemuxerTest do
```

Comment:
I think we need waaay more tests. Regarding the VINT and other stuff that is quite separate from other parts.